example/balances.prod.json

```diff
@@ -0,0 +1,8 @@
+{
+    "project": "example",
+    "env": "prod",
+    "name": "helloworld",
+    "tag": "production",
+    "task_role_arn": "example-prod-helloworld",
```

Comment:
are these supposed to be arns? maybe add an arn-compatible sring

Reply:
Changed to example ARNs; it looks like AWS supports both the name or ARN here, but when querying the service via describe-services, it always returns the ARN.